package/src/components/Channel/Channel.tsx

```diff
@@ -825,7 +825,7 @@ const ChannelWithContext = <
       setHasMore(true);
       copyChannelState();
     } catch (err) {
-      setError(err);
```

Comment:
Can you revert these changes? I was discussing with Khushal, and he is going to take care of this as part of https://stream-io.atlassian.net/browse/CRNS-434

Reply:
ok so the reason i updated TS in here its because I was using some v4.4 features in the cache interface typing to have `channel_item_${channelId}` as indexed key and in the types still force the keys to be `channel_item_${string}`. that can be replaced with just using a random string as indexed key, the bad part is that now users can get a little bit confused about which keys to expect in the cache interface.

after updating typescript, i noticed taht the err argument now has an `unknown` type, which is incompatible with the boolean type (it was any in the past so thats why it used to work).

i guess Khushal is going to face the same problems cause at some point we are going to update it anyways, so I would say we just update now, use booleans, and then Khushal fixes it by using the err object. Wha do you think?

Reply:
Coolio. Sounds good to me. @khushal87 please take a note, and get ready for some conflict :D

Reply:
Okay, I haven't started with the error ticket yet, so it's okay. GTG :rocket: